shardingsphere-features/shardingsphere-sharding/shardingsphere-sharding-route/src/test/java/org/apache/shardingsphere/sharding/route/engine/condition/generator/impl/ConditionValueBetweenOperatorGeneratorTest.java

```diff
@@ -36,11 +37,11 @@
 import static org.junit.Assert.assertTrue;
 
 public final class ConditionValueBetweenOperatorGeneratorTest {
-    
+
```

Comment:
A newline is supposed to start with 4 blank spaces. Just a little issue of check style. Apart from that, I approved. Thanks for the clean coding!